app/components/ranking.jsx

```diff
@@ -52,14 +54,15 @@ export default function Ranking(props) {
     >
       {responseOptions.map(option => {
         return (
-          <label>
+          <label key={option}>
             <input
               disabled={disabled || false}
               checked={response === option}
               type="radio"
               value={option}
               name={`importance-${option}`}
               className={cx(styleClasses.hideDefaultRadio, `ranking${option}`)}
+              onChange={() =>onSelectionChange(option)}
```

Comment:
This is good. I understand why there was the warning.  Please remove the other onChange line that's in the Div above, so we don't get 2 calls to onchange for ever change.

Reply:
Hello @ddfridley , I have removed onChange() Handler from the "div" tag after removing it I couldn't select the option. To resolve the selected option I try to pass  onSelectionChange function directly as the event handler. let me know any changes